libfuzzer/FuzzerLoop.cpp

```diff
@@ -183,6 +183,9 @@ void Fuzzer::DumpCurrentUnit(const char *Prefix) {
     PrintHexArray(CurrentUnitData, UnitSize, "\n");
     PrintASCII(CurrentUnitData, UnitSize, "\n");
   }
+  if (EF->LLVMFuzzerCustomOutput) {
+    EF->LLVMFuzzerCustomOutput(CurrentUnitData, CurrentUnitSize);
```

Comment:
I think it makes sense to call custom user code last, after the test case has been written to a file, just in case the custom user output code panics again or something like that.

Reply:
Works. I'm going to be submittign this patch upstream anyway.